code/2_analysis/maps/fig1.R

```diff
@@ -0,0 +1,63 @@
+# load grace data
+grace <- read_dta("data/input/grace/grace.dta")
+
+# restrict to arable land and cells with valid delta_TWS data
+grace <- grace %>%
```

Comment:
we use this a lot, seems like it could be a function but not a big deal just a suggestion